harness/determined/common/api/request.py

```diff
@@ -42,6 +42,19 @@ def maybe_upgrade_ws_scheme(master_address: str) -> str:
         return master_address
 
 
+def make_interactive_task_url(
+    task_id: str, service_address: str, description: str, resource_pool: str, task_type: str
+) -> str:
+    wait_path_url = service_address + "/notebooks/{}/events".format(task_id)
+    wait_page_url = "/det/wait/{}/{}?eventUrl={}&serviceAddr={}".format(
+        task_type, task_id, wait_path_url, service_address
+    )
+    notebook_web_url = "/det/interactive/{}/{}/{}/{}/{}".format(
```

Comment:
nitpick, but could this variable be changed to `task_web_url` so there's no confusion about it being generic for our other task types